src/pages/LandingPage/MapComponent.js

```diff
@@ -1,77 +1,71 @@
-import React from 'react';
-import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
-import L from 'leaflet';
-import PropTypes from 'prop-types';
-import originIcon from 'App/assets/images/origin.svg';
-import destinationIcon from 'App/assets/images/destination.svg';
+import React, { useEffect, useState } from "react";
+import { MapContainer, TileLayer } from "react-leaflet";
+import L, { Icon } from "leaflet";
+import PropTypes from "prop-types";
+// import originIcon from "App/assets/images/origin.svg";
+// import destinationIcon from "App/assets/images/destination.svg";
+import busStopsData from "./busStopData.json";
+import busStopIcon from "../../App/assets/images/bus-stop6.svg";
 
+import icon from 'leaflet/dist/images/marker-icon.png';
+import iconShadow from 'leaflet/dist/images/marker-shadow.png';
+import iconRetina from 'leaflet/dist/images/marker-icon-2x.png';
 
-const MapComponent = (props) => {
-  const samplePosition = {
-    lat: -1.9297648,
-    lng: 29.9871554
-  };
 
-  if (props.origin !== undefined && props.destination !== undefined) {
-    const origin = {
-      ...props.origin
-    };
 
-    const destination = {
-      ...props.destination
-    };
+const stopIcon = new Icon({
+  iconUrl: busStopIcon,
+  iconSize: [60, 70],
+});
 
-    const origIcon = L.icon({
-      iconUrl: originIcon,
-      iconSize:     [38, 95],
-      shadowSize:   [50, 64],
-      iconAnchor:   [22, 94],
-      shadowAnchor: [4, 62],
-      popupAnchor:  [-3, -76]
+const MapComponent = () => {
+  function onEachFeature(feature, layer) {
+    layer.setIcon(stopIcon);
+    layer.bindPopup(`<p>Name: ${feature.properties.name}</p>`);
+  }
```

Comment:
Change this to an arrow function

Reply:
I changed the above function to arrow function